### ISSUE TITLE
Please attribute the original source

### ISSUE THREAD

**Issue by SlayterDev** [doged/DOGED-iOS#1]
If you're going to fork from my wallet to make a new one, please at least attribute me as the original author. Thank you.

**Comment by doged** [comment]
of course, and your name is in most of the files from what i see. any idea on how i can add ntime, because i cant seem to parse incoming tx's properly. this hasnt been released yet, but once its finished i plan on doing a PR on coinami and fixing up the read me with more detailed instructions/credits.

**Comment by doged** [comment]
or where i can add nTime rather.. thats what i needed to do in the electrum client, because although its disabled currently, the sauce for doged has PoS in it, as a "just in case".

**Comment by doged** [comment]
of course. i am going to update the readme with that and compilation instructions for xcode, how to install, etc... i havent released this yet.. just getting around to compiling it right now..

**Comment by doged** [comment]
hey @SlayterDev only issue im getting on compile is NMSSH.h not found, in senddogecontroller.h, i believe it has something to do with how the git directory structure is laid out for nmssh.. any suggestions? also i added your author name to the readme.. you know opalcoin forked this as well right? that is where i found it originally.
